packages/ndla-ui/src/model/Transcriptions.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+export interface Transcription {
+  trad?: string;
```

Comment:
Er trad kort for traditional? Er dette noe du har funnet på eller er det sånn det er skrevet i backend? Visst ikke noen av de kanskje vi skal ta å skrive `traditional`? Er ikke så fan av å forkorte navn 😄

Reply:
Den er hentet fra backend.

Reply:
Backend har en map med strenger, så dette kan like gjerne være "traditional". La ikkje inn føringer på dette der.